src/Microsoft.VisualStudio.ProjectSystem.Managed/ValueTaskExtension.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
Seems like this came from Roslyn. We should include a link to where it was sourced from to make it easier to keep up to date over time.

Reply:
It's not an exact copy, I prefer to remove the extra headline,,,